WordPress/src/main/res/layout/help_activity.xml

```diff
@@ -87,6 +87,44 @@
                     android:text="@string/wordpress_help_center_button"
                     android:visibility="gone"/>
 
+                <LinearLayout
+                    android:id="@+id/JpFaqContainer"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="?android:attr/selectableItemBackground"
+                    android:orientation="vertical"
+                    android:padding="@dimen/margin_extra_large"
+                    android:visibility="gone"
+                    tools:visibility="visible">
+
+                    <ImageView
+                        android:id="@+id/wp_jp_logo"
+                        android:layout_width="@dimen/margin_64dp"
+                        android:layout_height="@dimen/margin_64dp"
+                        android:contentDescription="@string/support_faq_caption"
+                        android:src="@drawable/ic_wordpress_jetpack_logo" />
+
+                    <com.google.android.material.textview.MaterialTextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/support_faq_title"
+                        android:textAppearance="?attr/textAppearanceSubtitle1" />
+
+                    <com.google.android.material.textview.MaterialTextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/support_faq_caption"
+                        android:textAppearance="?attr/textAppearanceCaption" />
+                </LinearLayout>
+
+                <View
+                    android:id="@+id/JpFaqContainerBottomDivider"
+                    android:visibility="gone"
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/divider_size"
+                    android:background="?android:attr/listDivider"
+                    tools:visibility="visible"/>
```

Comment:
np: Can we move this divider into `JpFaqContainer`? This way, we won't have to manage the divider's visibility as separately from `JpFaqContainer`'s visibility.

Reply:
There's a margin on the parent view group, it wouldn't extend to the edges if it is inside.  We could remove the margin and add it to the child views, and then it just adds more lines!